block.go

```diff
@@ -1243,11 +1265,18 @@ gatherlines:
 
 		// calculate the indentation
 		indent := 0
-		for indent < 4 && line+indent < i && data[line+indent] == ' ' {
-			indent++
+		indentIndex := 0
+		if data[line] == '\t' {
+			indentIndex++
+			indent += 4
+		} else {
+			for indent < 4 && line+indent < i && data[line+indent] == ' ' {
+				indent++
+				indentIndex++
+			}
 		}
```

Comment:
Is it neccessary to put `for` loop inside `else` block, or would the behavior be the same if `for` block would simply follow the `if` statement?

The reason I ask is because it looks a little inconsistent with line 1178/1200 (beginning of `parser.listItem` method), and I wonder if we should make it consistent (either by moving that `for` into `else`, or by moving this `for` out of the `else`).